### ISSUE TITLE
Wrong react file upload code

### ISSUE THREAD

**Issue by joubeh** [inertiajs/inertiajs.com#375]
The react code in page "https://inertiajs.com/file-uploads" for uploading files using useForm hook has error.
`<input type="file" value={data.avatar} onChange={e => setData('avatar', e.target.files[0])} />`
The code `value={data.avatar}` should be removed. it gets error because: `InvalidStateError: Failed to set the 'value' property on 'HTMLInputElement': This input element accepts a filename, which may only be programmatically set to the empty string.`

**Comment by Jussi-Mannisto** [comment]
The attribute `value={data.avatar}` causes the issue, and removing it resolves it. The `value` attribute of a file input cannot be programmatically modified, except by setting it to an empty string.

**Comment by JsExpertCoder** [comment]
yes @joubeh, and you are right @Jussi-Mannisto , I also notice that, and I submitted a PR: https://github.com/inertiajs/inertiajs.com/pull/396

**Comment by pascalbaljet** [comment]
Fixed in PR #396.
